test/xpu_api/iterators/predef.iterators/move.iterators/move.iter.ops/move.iter.op.const/convert.pass.cpp

```diff
@@ -70,27 +68,26 @@ kernel_test()
                 ret_access[0] &= test<random_access_iterator<const Base*>>(random_access_iterator<Derived*>(&d));
                 ret_access[0] &= test<Base*>(&d);
 
-#if TEST_STD_VER > 14
                 {
                     constexpr const Derived* p = nullptr;
-                    constexpr s::move_iterator<const Derived*> it1 = s::make_move_iterator(p);
-                    constexpr s::move_iterator<const Base*> it2(it1);
+                    constexpr dpl::move_iterator<const Derived*> it1 = dpl::make_move_iterator(p);
+                    constexpr dpl::move_iterator<const Base*> it2(it1);
                     static_assert(it2.base() == p);
                 }
-#endif
             });
         });
     }
     return ret;
 }
+#endif // TEST_DPCPP_BACKEND_PRESENT
 
 int
-main(int, char**)
+main()
 {
+#if TEST_DPCPP_BACKEND_PRESENT
     auto ret = kernel_test();
-    if (ret)
-        std::cout << "Pass" << std::endl;
-    else
-        std::cout << "Fail" << std::endl;
-    return 0;
+    EXPECT_TRUE(ret, "Wrong result of iterator check #1 in kernel_test()");
```

Comment:
=> "Wrong result of move_iterator check with copy constructor in kernel_test()"

Reply:
Fixed.